tests/components/holiday/test_calendar.py

```diff
@@ -0,0 +1,88 @@
+"""Tests for calendar platform of Holiday integration."""
+
+from datetime import datetime, timedelta
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.holiday import calendar
+from homeassistant.components.holiday.const import CONF_PROVINCE, DOMAIN
+from homeassistant.const import CONF_COUNTRY
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.util import dt as dt_util
+
+from tests.common import MockConfigEntry
+
+
+async def assert_setup_calendar(hass: HomeAssistant) -> MockConfigEntry:
+    """Set up the calendar."""
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={CONF_COUNTRY: "CH", CONF_PROVINCE: "GE"},
+        title="Switzerland, GE",
+    )
+    config_entry.add_to_hass(hass)
+
+    await hass.config_entries.async_setup(config_entry.entry_id)
+    await hass.async_block_till_done()
+
+    return config_entry
+
+
+async def test_holiday_calendar_entity(
```

Comment:
We should have a test for a non default language. For example setup France with English (see my previous comment regarding "en" vs. "en_US")

Reply:
Done.